fl4health/model_bases/apfl_base.py

```diff
@@ -69,5 +71,7 @@ def update_alpha(self) -> None:
         self.alpha = alpha
 
     def layers_to_exchange(self) -> List[str]:
-        layers_to_exchange: List[str] = [layer for layer in self.state_dict().keys() if "global_model" in layer]
+        layers_to_exchange: List[str] = [
+            layer for layer in self.state_dict().keys() if layer.startswith("global_model.")
```

Comment:
Changing this function to mirror FENDA and FedPer. In particular, we only want the layer name to start with `global_model.` rather than have it appear anywhere else in a name.